### ISSUE TITLE
Next: cards: share button cannot be empty, and z-index issue

### ISSUE THREAD

**Issue by Fupete** [italia/designers.italia.it#399]
![immagine](https://user-images.githubusercontent.com/6736103/199052049-f7f2a702-107e-4760-bdf4-b2d173eb4661.png)

There are two issues on this card share button:
1. the share button cannot be empty for accessibility, we need to add one of the correct a11y approaches (eg. visual hidden text, aria, ...)
2. the z-index issue, when a card is near another one on the right, needs to be addressed.

cc @astagi @OscarBresolin

**Comment by Fupete** [comment]
Fixed here https://github.com/italia/designers.italia.it/commit/ee6408e0004e6905c676824f89058710cfa74172 and here https://github.com/italia/designers.italia.it/commit/57c6e5d5a07c718ede08109a6150f08e991dbaf0
